src/main/resources/application.properties

```diff
@@ -116,18 +116,19 @@ management.endpoint.prometheus.enabled=true
 
 # Mail
 # sender for all emails
-# uv.mail.from=sender@urlaubsverwaltung.test
+uv.mail.sender=absender@urlaubsverwaltung.cloud
```

Comment:
Diese Änderungen müssen im Wiki dokumentiert werden unter https://github.com/synyx/urlaubsverwaltung/wiki/Urlaubsverwaltung-3.0-Migration-Guide

Reply:
Wo/wie wurden bisher die Mail Properties konfiguriert?

Reply:
Das war wohl nur ein Tippfehler in dem auskommentierten ding.